### PR TITLE
Improved the External Data page introduction content

### PR DESCRIPTION
**Pull request by benji-glitsos-ga**
* [x] I have spellchecked my content using Microsoft Word, Grammarly, or otherwise.
* [ ] If required, update the [DEA Tech Alerts and Changelog][TechAlertsChangelog] and the [DEA Sandbox Updates banner][SandboxUpdatesBanner].

[TechAlertsChangelog]: https://github.com/GeoscienceAustralia/dea-knowledge-hub/blob/main/docs/tech-alerts-changelog/_tech_alerts_changelog.md
[SandboxUpdatesBanner]: https://bitbucket.org/geoscienceaustralia/datakube-apps/src/64c28bbf3d0e019d8940547a22f78b9bfd58d739/clusters/dea-sandbox/sandbox.yaml

Rewording content.

### CODE REVIEW

**Comment by robbibt** [comment]
I think the new wording might be a little too wordy, and I liked the text starting with a more direct description of the data contained in this section. Could something like this work?

> DEA provides access to selected external data to assist with analysing and validating DEA products. These external data products are not produced by DEA, and are provided without modifications except when required for compatibility with our systems.

**Comment by benji-glitsos-ga** [comment]
@robbibt Your rewording sounds good. I edited it slightly and pushed to the branch.

**Review by robbibt — APPROVED** [review]
